src/calculator/index.ts

```diff
@@ -108,13 +109,14 @@ type RawRound = {
   id: string;
 };
 
-export default class Calculator {
+export default class  {
```

Comment:
can we put back the name just to have it more explicit?

Reply:
🐙 oops accidentally deleted that! yup!